app/views/studies/_shared_sync_functions.js.erb

```diff
@@ -1,9 +1,81 @@
 // shared JS functions for sync forms
 
-$('#study-file-<%= study_file.id %>').on('change', '.file-type', function() {
+// Used for sync CSFV
+if (typeof window.SCP.readOnlyToken === 'undefined') {
+  window.SCP.readOnlyToken = '<%= get_read_access_token(@study, current_user) %>'
+}
+
+/**
+* Transform validation results to HTML that can be shown to user.
+*
+* This function is a temporary backport of the issue messaging from
+* FileUploadControl.js.  It should be replaced when sync code migrates to
+* React.
+*/
+function getIssuesHtml(results, fileName, fileId) {
+  let issuesHtml = ''
+
+  const errorMsgs = results.errors.map(error => error[2])
+  const warningMsgs = results.warnings.map(warning => warning[2])
+
+  if (errorMsgs.length > 0) {
+    // Tell user how to get in touch
+    const supportEmailAddress = 'scp-support@broadinstitute.zendesk.com'
+    const supportEmailLink =
+      `<a href="mailto:${supportEmailAddress}">` +
+        supportEmailAddress +
+      '</a>'
+    const supportMessage =
+      '<div>' +
+        `If you need help, let us know at ${supportEmailLink}.  ` +
+        'Please include the above errors, your file, and study accession.' +
```

Comment:
I like this inclusion of a help link right by the errors, let's add this to the new ValidationMessages component.  Since there's already likely to be a lot of text on the screen, looking for a way to shorten this might be helpful
`Need help?  Send us an email at {email} with the errors, your accession (${studyAccession}), and a description/attachment of your file`

Reply:
Nicely trimmed, and reasonable to scope as part of a separate task for `ValidationMessage`.

Reply:
If you prefer your text, keep it (any text refinement suggestions from me should always be taken with grains of salt).  But otherwise, at least make the text change here and now so we remember and can iterate on it later with users having seen it

Reply:
Done!  I included it in `ValidationMessage`.